test/unit/mock/objects.js

```diff
@@ -587,6 +587,84 @@ const installationMock = {
   uid: 'installation_uid'
 }
 
+const globalFieldDiff = [{
+  "uid": "global_field_uid",
+  "title": "Global Field Title",
+  "type": "global_field",
+  "status": "base_only"
+}]
+
+const contentTypeDiff = [{
+  "uid": "content_type_uid_1",
+  "title": "Content Type 1 Title",
+  "type": "content_type",
+  "status": "compare_only"
+},
+{
+  "uid": "content_type_uid_2",
+  "title": "Content Type 2 Title",
+  "type": "content_type",
+  "status": "modified"
+}]
+
+const branchCompareAllMock = {
+  "branches": {
+    "base_branch": "UID",
+    "compare_branch": "dev"
+  },
+  "diff": [...globalFieldDiff, ...contentTypeDiff],
+  "next_url":"https://api.contentstack.io/v3/stacks/branches/compare?base_branch=main&compare_branch=dev&skip=0&limit=100"
+}
+
+const branchCompareContentTypeMock = {
+  "branches": {
+    "base_branch": "UID",
+    "compare_branch": "dev"
+  },
+  "diff": [ ...contentTypeDiff ],
+  "next_url":"https://api.contentstack.io/v3/stacks/branches/compare?base_branch=main&compare_branch=dev&skip=0&limit=100"
+}
+
+const branchCompareGlobalFieldMock = {
+  "branches": {
+    "base_branch": "UID",
+    "compare_branch": "dev"
+  },
+  "diff": [...globalFieldDiff],
+  "next_url":"https://api.contentstack.io/v3/stacks/branches/compare?base_branch=main&compare_branch=dev&skip=0&limit=100"
+}
+
+const branchMergeAllMock = {
+  "uid": "UID",
+  "stack": "blt736475345hsgd22",
```

Comment:
@nadeem-cs always run `git secrets --scan-history` before committing code to avoid tokens to be committed on github.
Remove these tokens from history as well for this branch and commit.